mimetype_test.go

```diff
@@ -597,3 +597,10 @@ func TestExtend(t *testing.T) {
 		})
 	}
 }
+
+func TestTzif(t *testing.T) {
+	mtype, _ := DetectFile("testdata/tzif.txt")
+	if mtype.Is("application/tzif") {
+		t.Fatal("simple text file should not be detected as Tzif file")
+	}
+}
```

Comment:
Please remove this test function and the file at testdata/tzif.txt. I added it to better show you the problem and it is not needed anymore.

Reply:
Removed in commit.